db_stress_tool/db_stress_common.h

```diff
@@ -235,6 +235,7 @@ DECLARE_bool(in_place_update);
 DECLARE_string(memtablerep);
 DECLARE_int32(prefix_size);
 DECLARE_bool(use_merge);
+DECLARE_uint32(use_put_entity_one_in);
```

Comment:
Just to double check, can you build with `ROCKSDB_NO_FBCODE=1` to make sure it compiles?

Reply:
Yep, `ROCKSDB_NO_FBCODE` seems to be working just fine.